### ISSUE TITLE
installation error

### ISSUE THREAD

**Issue by MadMan247** [YunoHost-Apps/immich_ynh#31]
### Describe the bug

Installation error

### Context

- Hardware: Raspberry Pi 4 B (4GB) at home
- YunoHost version: 11.2.11.2
- I have access to my server: Through SSH | through the webadmin
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: no
- Using, or trying to install package version1.101.0~ynh1

### Steps to reproduce

-click "install"
-error

### Expected behavior

-click "install"
-installs

### Logs

https://paste.yunohost.org/raw/foyevefaxi

**Comment by MadMan247** [comment]
different attempts;

Most recent attempt:
https://paste.yunohost.org/raw/exucuqofoj
![image](https://github.com/YunoHost-Apps/immich_ynh/assets/54383546/d762b7e1-53ad-4372-8402-ab3e74eb5629)
stack, just in case it can provide additional information

**Comment by ewilly** [comment]
I can't understand why it failing...
Is it still failing ?
If yes, can you retry the install with that command `sudo yunohost app install https://github.com/YunoHost-Apps/immich_ynh --debug --no-remove-on-failure` and share the log ?

**Comment by MadMan247** [comment]
Absolutely! Thank you for your attention to this matter.
Upon executing that command, I have run into the same error. [Here are the logs](https://paste.yunohost.org/raw/iyadibayat)
attempting to go to the URL that I installed immich under results in the Nginx welcome page
![image](https://github.com/YunoHost-Apps/immich_ynh/assets/54383546/78409d4b-ce00-4ff7-b2f1-1215d1c1c23d)

and the /admin goes to my yunohost SSO, and then to an error 404 page
![image](https://github.com/YunoHost-Apps/immich_ynh/assets/54383546/f074d532-641c-4a42-ae1a-3f75e0d448e1)


While I did answer "no" for "Are you in a special context or did you perform some particular tweaking on your YunoHost instance" in the initial issue, I may have modified something unintentionally. I will be able to perform a clean installation attempt in a few days, as I will be nuking my current yunohost setup.

**Comment by ewilly** [comment]
Ok as you want otherwise you can share with me the content of `/var/www/immich/env` but in private message as there is the psql password in it.

**Comment by MadMan247** [comment]
I am... not sure how to private message on Github. I can find no way to send a message on your profile. Regardless, looking at the env file, I do not see a visible password, but instead some placeholder tiles such as `__DB_PORT__`,` __INSTALL_DIR__`, etc. For this reason, I feel comfortable posting it here. Just in case, I will replace the "password" (it looks like a placeholder above) with asterisks.
```
## General
NODE_ENV=production
IMMICH_MEDIA_LOCATION=/home/yunohost.app/immich
IMMICH_WEB_ROOT=__INSTALL_DIR__/app/www
IMMICH_REVERSE_GEOCODING_ROOT=__INSTALL_DIR__/app/resources

## Ports
SERVER_PORT=__PORT__
MICROSERVICES_PORT=__PORT_MICROSERVICES__
MACHINE_LEARNING_HOST=127.0.0.1
MACHINE_LEARNING_PORT=__PORT_MACHINELEARNING__
IMMICH_MACHINE_LEARNING_URL=http://127.0.0.1:__PORT_MACHINELEARNING__

## Database
DB_HOSTNAME=127.0.0.1
DB_PORT=__DB_PORT__
DB_USERNAME=immich
DB_PASSWORD=_********_
DB_DATABASE_NAME=immich
DB_VECTOR_EXTENSION=pgvector

## Redis
REDIS_HOSTNAME=127.0.0.1
```

**Comment by ewilly** [comment]
hum the issue is not on the package but on your system some placeholder were replaced but other not which is related to an helper that does not work as it should... You should encounter the problem with some other app...
I will close the issue and can only suggest you to repair your system.

**Comment by MadMan247** [comment]
I see. Thank you for your time, I will be sure to let you know my progress when I attempt to install it on a clean installation.

As closing remarks, I have not encountered this specific issue (task S was not terminated) on other apps, but I have had more installations fail than I would have expected, so it may still be related.
I have not made many modifications to my system myself, so I suspect a different app caused this.

Thank you again for your time, effort, and dedication. Your efforts mean a lot to a lot of people.

**Comment by MadMan247** [comment]
New day, my fresh new installation, new error.

Order of operations:
Fresh Install (put yunohost image on medium)
create default domain/user
update all packages
create domain imgs.example.com
install immich on imgs.example.com

https://paste.yunohost.org/raw/vekokayinu

attempting again with the command `sudo yunohost app install https://github.com/YunoHost-Apps/immich_ynh --debug --no-remove-on-failure` resulted in the original error. Please see: https://paste.yunohost.org/raw/wekabazoye

Because of this, I can confidently say that it was neither my own misconfiguration, nor the misconfiguration of another application. I haven't even assigned a let's encrypt certificate to this yunohost instance.

I hope that you consider reopening this issue.

```
 /var/www/immich/env:

## General
NODE_ENV=production
IMMICH_MEDIA_LOCATION=/home/yunohost.app/immich
IMMICH_WEB_ROOT=__INSTALL_DIR__/app/www
IMMICH_REVERSE_GEOCODING_ROOT=__INSTALL_DIR__/app/resources

## Ports
SERVER_PORT=__PORT__
MICROSERVICES_PORT=__PORT_MICROSERVICES__
MACHINE_LEARNING_HOST=127.0.0.1
MACHINE_LEARNING_PORT=__PORT_MACHINELEARNING__
IMMICH_MACHINE_LEARNING_URL=http://127.0.0.1:__PORT_MACHINELEARNING__

## Database
DB_HOSTNAME=127.0.0.1
DB_PORT=__DB_PORT__
DB_USERNAME=immich
DB_PASSWORD=_********_
DB_DATABASE_NAME=immich
DB_VECTOR_EXTENSION=pgvector

## Redis
REDIS_HOSTNAME=127.0.0.1

```

**Comment by ewilly** [comment]
still not an issue due to immich, something is wrong on your configuration. It fails when using a common helper script provide by yunohost not by immich.
If you look your log there is for example those messages you have to take care :
`2024-04-24 05:39:16,995: DEBUG - Wi-Fi is currently blocked by rfkill.
2024-04-24 05:39:16,996: DEBUG - Use raspi-config to set the country before use.`
you could have  loot at those link that can help :
[https://forum.yunohost.org/t/problem-by-installing-yunohost/11384/26](https://forum.yunohost.org/t/problem-by-installing-yunohost/11384/26)
[https://github.com/YunoHost-Apps/gitea_ynh/issues/61](https://github.com/YunoHost-Apps/gitea_ynh/issues/61)
If you don't find opening a post on the forum could help...

**Comment by MadMan247** [comment]
Hello. Thank you for taking the time to respond. I apologize for taking so long to get back to you. I appreciate your time.

Personally, I am very confused right now. It makes no sense that the wifi being disabled, and no wan country being set would have such an impact on the success of the install. As far as I can tell, that is all that I changed V.S. the last time that I tried. Maybe some of the updates you pushed this last week solved this issue.  In addition to setting the WAN country, I had also set a time zone.

Well, it works now. Cheers.

I really again have to emphasize that it did not make sense to me to set that up prior, given that I use ethernet. I'd go as far as to say that the wan-country being unset being the issue is such a silly, silly, ridiculously silly problem, it's probably an unintended issue, or something that should be otherwise resolved as to prevent future fuss. To my knowledge, this has not caused me issues in any other applications.

Thank you again for your diligence.
